src/polkitmateauthenticationdialog.c

```diff
@@ -791,7 +791,7 @@ polkit_mate_authentication_dialog_constructed (GObject *object)
   rows = 0;
   if (dialog->priv->details != NULL)
     {
-      guint n;
+      gint n;
```

Comment:
Why change that one?  AFAICT it's only used to iterate over `keys`, so either sign is fine (we could argue forever, but I'd keep it as it is unless necessary)

Reply:
I don't have access rights for this repo, feel free to update this PR, otherwise you can close it.

Reply:
> I don't have access rights for this repo, feel free to update this PR, otherwise you can close it.

You deleted you access to MATE repositories for yourself. No Problem, we can fix this mistake/issue very quick.